skfem/assembly/form/bilinear_form.py

```diff
@@ -18,9 +18,9 @@ class BilinearForm(Form):
     trial function ``u``, test function ``v``, and a dictionary of additional
     parameters ``w``.
 
-    >>> from skfem import BilinearForm, InteriorBasis, MeshTri, ElementTriP1
+    >>> from skfem import BilinearForm, Basis, MeshTri, ElementTriP1
     >>> form = BilinearForm(lambda u, v, _: u * v)
-    >>> form.assemble(InteriorBasis(MeshTri(), ElementTriP1())).todense()
+    >>> form.assemble(Basis(MeshTri(), ElementTriP1())).todense()
```

Comment:
Maybe `toarray`?  [`.todense`](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.spmatrix.todense.html#scipy.sparse.spmatrix.todense) returns a [`numpy.matrix`](https://numpy.org/devdocs/reference/generated/numpy.matrix.html#) and

> It is no longer recommended to use this class, even for linear algebra. Instead use regular arrays. The class may be removed in the future.